docs/troubleshooting.md

```diff
@@ -0,0 +1,48 @@
+# Issues we met
+
+This document collects issues that we had with sssd-test-suite and their
+solution.
+
+## Ad machine sudden shutdown
+Probably license expired. Run
+
+    ./sssd-test-suite up -s ad ad-child && ./sssd-test-suite provision rearm
+
+
+## TASK [host : Install required vagrant plugins] failed due to vagrant dependencies
```

Comment:
This happens because we are mixing vagrant plugins through gems distribution with plugins packaged with rpm. Its better to ensure that no vagrant plugins are installed through rpms.